### ISSUE TITLE
Why is the input image not normalized?

### ISSUE THREAD

**Issue by Sumching** [prlz77/vgg-face.pytorch#3]
Hello,
I noticed the input image is not normoalized to range [-1, 1], has the model trained with data that is not in this range?.  And is double precision required?

**Comment by prlz77** [comment]
Exact, they trained the model with mean subtraction only in the 255 range.
It should work if you cast it to Float32.
